bin/pana.dart

```diff
@@ -32,12 +32,9 @@ final _parser = ArgParser()
       help: 'The server that hosts <package>.', defaultsTo: defaultHostedUrl)
   ..addOption('line-length',
       abbr: 'l', help: 'The line length to use with dartfmt.')
-  ..addOption('verbosity',
-      help: 'Configure the details in the output.',
-      allowed: ['compact', 'normal', 'verbose'],
-      defaultsTo: 'normal')
-  ..addFlag('scores', help: 'Include scores in the output JSON.')
-  ..addFlag('report', help: 'Print markdown report instead of JSON.')
+  ..addFlag('hosted', help: 'Shortcut to `--source hosted`.')
```

Comment:
Can we make this one be the default? And hide the  '--source' option, just keeping it for backwards compatibility?